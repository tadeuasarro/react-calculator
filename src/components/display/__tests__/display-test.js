import React from 'react';
import renderer from 'react-test-renderer';
import { render, screen } from '@testing-library/react';
import '@testing-library/jest-dom';
import Display from '../display';

describe('Display', () => {
  it('renders correctly', () => {
    const tree = renderer.create(<Display />).toJSON();
    expect(tree).toMatchSnapshot();
  });

  it('creates a Display component', () => {
    render(<Display result="0" />);
    const element = screen.getByText('0');
    expect(element).toBeInTheDocument();
  });
});
