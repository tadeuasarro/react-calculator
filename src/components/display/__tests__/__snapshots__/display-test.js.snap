// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Display renders correctly 1`] = `
<div
  id="display"
>
  0
</div>
`;

exports[`Dispplay renders correctly 1`] = `
<div
  id="display"
>
  0
</div>
`;
